src/service/action-impl.js

```diff
@@ -534,15 +535,6 @@ export class ActionService {
   invoke_(invocation) {
     const {method, tagOrTarget} = invocation;
 
-    // Check that this action is whitelisted (if a whitelist is set).
-    if (this.whitelist_) {
-      if (!isActionWhitelisted_(invocation, this.whitelist_)) {
-        this.error_(`"${tagOrTarget}.${method}" is not whitelisted ${
-          JSON.stringify(this.whitelist_)}.`);
-        return null;
-      }
-    }
```

Comment:
Moving this to `base-element.js` will break whitelisting for global actions, e.g. `AMP.setState`.